doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -1048,8 +1064,10 @@ Platform and Access (source `RM 7.9.2 <../../../ref_model/chapters/chapter07.md#
 Table 2-7: Reference Model Requirements - Platform and Access
 Requirements
 
-Confidentiality and Integrity (source `RM 7.9.3 <../../../ref_model/chapters/chapter07.md#793-confidentiality-and-integrity>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Confidentiality and Integrity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+(source :ref:`ref_arch/openstack/chapters/chapter02:confidentiality and integrity`)
```

Comment:
error, it should be:
(source :ref:`ref_arch/openstack/chapters/chapter07:confidentiality and integrity`)